frontend/public/data/rbd/ch/admin1_2023_02_01.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Are you sure we want to keep these "empty" files?

Reply:
Or is it to match what date input we provide somewhere else?

Reply:
these were automatically generated using a script. I think it might raise an error if we remove it as the admin 2 json for the same date is used in the layer and 'falls back' on admin 1.